CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## Unreleased
+
+## v0.8.2 (2025-05-04)
```

Comment:
Bit awkward with release versioning where we have both faucet and node.

Probably we can just.. keep them in sync regardless of changes for now.

We would need to change all version numbers if we really care about this now I guess.